{% if cookiecutter.library_prefix %}{{ cookiecutter.library_prefix | capitalize }}_{% endif %}CircuitPython_{{ cookiecutter.library_name}}/.pre-commit-config.yaml

```diff
@@ -17,3 +17,17 @@ repos:
     -   id: check-yaml
     -   id: end-of-file-fixer
     -   id: trailing-whitespace
+-   repo: local
+    hooks:
+    -   id: "pylint-library"
+        name: "pylint-library"
+        description: Run pylint rules on library python code files
+        entry: /usr/bin/env bash -c
+        args: ['pylint $( find . -path "./adafruit*.py" )']
```

Comment:
Not all libraries start with the adafruit

Reply:
Very true. I think I'd copied my original from a core/Adafruit library, and that pattern "stuck". I'll update for the correct pattern, excluding the setup.py, examples/*, etc